apps/bfd-model/bfd-model-rif/src/main/java/gov/cms/bfd/model/rif/schema/DatabaseSchemaManager.java

```diff
@@ -89,6 +89,7 @@ public static void dropIndexes(DataSource dataSource) {
         placeholders.put("logic.index-create-concurrently", "");
         placeholders.put("logic.sequence-start", "start with");
         placeholders.put("logic.sequence-increment", "increment by");
+        placeholders.put("logic.non-hsql-only-line", "--");
```

Comment:
This name makes my head hurt. Could we just call it the `postgresql-only-line`?

Reply:
sure, although that's also confusing since there's no mention of postgres anywhere else.